### PR TITLE
Update: Revamped Documentation, moved from iframe to individual pages.

### PR DESCRIPTION
**Pull request by varun13169**
Update: Revamped Documentation, moved from iframe to individual pages.
            Add Support Prism JS for responsive documentation.
            Add home screen with animated logo.

### CODE REVIEW

File: components/badge/badgePage.html [hunk 1]
```diff
@@ -0,0 +1,190 @@
...
+            </div>
+            <!-- Header Component Ends -- Standard -->
+
+        </section>
+        <section class="container__sidebar">
+            <!-- Drawer Component Regular -->
+            <div class="drawer">
+                <div class="drawer__heading">
+                    <h3 class="drawer__title">Components</h3>
+                    <!-- <h3 class="drawer__subtitle">Component Lists</h3> -->
```
Comment (zoef-yt) [inline components/badge/badgePage.html]: remove commented code.

File: components/badge/badgePage.js [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+    })
+})
+
+function setDefault() {
+    let eleIdx = 0;
+    componentHolderEles.item(eleIdx).style.backgroundColor = `var(--primary-color-p2-lgt-10pct)`;
+    code.src = ifr[eleIdx];
+}
+
+// setDefault();
```
Comment (zoef-yt) [inline components/badge/badgePage.js]: remove commented code.

File: components/button/buttonPage.html [hunk 3]
```diff
@@ -0,0 +1,183 @@
...
+            </div>
+            <!-- Header Component Ends -- Standard -->
+
+        </section>
+        <section class="container__sidebar">
+            <!-- Drawer Component Regular -->
+            <div class="drawer">
+                <div class="drawer__heading">
+                    <h3 class="drawer__title">Components</h3>
+                    <!-- <h3 class="drawer__subtitle">Component Lists</h3> -->
```
Comment (zoef-yt) [inline components/button/buttonPage.html]: remove commented code.

File: components/button/buttonPage.html [hunk 4]
```diff
@@ -0,0 +1,183 @@
...
+            
+            
+        </section>
+        <section class="container__footer">
+
+        </section>    
+    </div>
+
+
+    <!-- <script src="index.js"></script> -->
```
Comment (zoef-yt) [inline components/button/buttonPage.html]: remove commented code.

File: components/card/cardPage.css [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+    background-color: var(--primary-color-p2-lgt-30pct);
+}
+
+.container__components-holder > div {
+    flex-basis: 0;
+    flex-grow: 1;
```
Comment (zoef-yt) [inline components/card/cardPage.css:28]: short-hand can be used. just a suggestion

File: components/card/cardPage.html [hunk 6]
```diff
@@ -0,0 +1,117 @@
...
+            </div>
+            <!-- Header Component Ends -- Standard -->
+
+        </section>
+        <section class="container__sidebar">
+            <!-- Drawer Component Regular -->
+            <div class="drawer">
+                <div class="drawer__heading">
+                    <h3 class="drawer__title">Components</h3>
+                    <!-- <h3 class="drawer__subtitle">Component Lists</h3> -->
```
Comment (zoef-yt) [inline components/card/cardPage.html]: remove commented code.

File: components/card/cardPage.html [hunk 7]
```diff
@@ -0,0 +1,117 @@
...
+            
+        </section>
+
+        <section class="container__footer">
+
+        </section>    
+    </div>
+
+
+    <!-- <script src="index.js"></script> -->
```
Comment (zoef-yt) [inline components/card/cardPage.html]: remove commented code.

File: components/card/cardPage.js [hunk 8]
```diff
@@ -0,0 +1,40 @@
...
+    })
+})
+
+function setDefault() {
+    let eleIdx = 0;
+    componentHolderEles.item(eleIdx).style.backgroundColor = `var(--primary-color-p2-lgt-10pct)`;
+    code.src = ifr[eleIdx];
+}
+
+// setDefault();
```
Comment (zoef-yt) [inline components/card/cardPage.js]: remove commented code.

File: components/image/imagePage.html [hunk 9]
```diff
@@ -0,0 +1,117 @@
...
+            </div>
+            <!-- Header Component Ends -- Standard -->
+
+        </section>
+        <section class="container__sidebar">
+            <!-- Drawer Component Regular -->
+            <div class="drawer">
+                <div class="drawer__heading">
+                    <h3 class="drawer__title">Components</h3>
+                    <!-- <h3 class="drawer__subtitle">Component Lists</h3> -->
```
Comment (zoef-yt) [inline components/image/imagePage.html]: remove commented code.

File: components/image/imagePage.html [hunk 10]
```diff
@@ -0,0 +1,117 @@
...
+            
+        </section>
+
+        <section class="container__footer">
+
+        </section>    
+    </div>
+
+
+    <!-- <script src="index.js"></script> -->
```
Comment (zoef-yt) [inline components/image/imagePage.html]: remove commented code.

File: components/image/imagePage.js [hunk 11]
```diff
@@ -0,0 +1,40 @@
...
+    })
+})
+
+function setDefault() {
+    let eleIdx = 0;
+    componentHolderEles.item(eleIdx).style.backgroundColor = `var(--primary-color-p2-lgt-10pct)`;
+    code.src = ifr[eleIdx];
+}
+
+// setDefault();
```
Comment (zoef-yt) [inline components/image/imagePage.js]: remove commented code.

File: components/input/inputPage.css [hunk 12]
```diff
@@ -0,0 +1,43 @@
...
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+    background-color: var(--primary-color-p2-lgt-30pct);
+}
+
+.container__components-holder > div {
+    flex-basis: 0;
+    flex-grow: 1;
```
Comment (zoef-yt) [inline components/input/inputPage.css:27]: short-hand can be used.

File: components/input/inputPage.html [hunk 13]
```diff
@@ -0,0 +1,117 @@
...
+            </div>
+            <!-- Header Component Ends -- Standard -->
+
+        </section>
+        <section class="container__sidebar">
+            <!-- Drawer Component Regular -->
+            <div class="drawer">
+                <div class="drawer__heading">
+                    <h3 class="drawer__title">Components</h3>
+                    <!-- <h3 class="drawer__subtitle">Component Lists</h3> -->
```
Comment (zoef-yt) [inline components/input/inputPage.html]: remove commented code.

File: components/input/inputPage.html [hunk 14]
```diff
@@ -0,0 +1,117 @@
...
+            
+        </section>
+
+        <section class="container__footer">
+
+        </section>    
+    </div>
+
+
+    <!-- <script src="index.js"></script> -->
```
Comment (zoef-yt) [inline components/input/inputPage.html]: remove commented code.

File: home.css [hunk 15]
```diff
@@ -0,0 +1,50 @@
...
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+}
+
+.home-container .container__content svg {
+    width: 30rem;
+    height: 30rem;
+    /* margin:-60px 60px 60px -60px; */
```
Comment (zoef-yt) [inline home.css]: remove commented code.

File: index-legacy.html [hunk 16]
```diff
@@ -0,0 +1,128 @@
...
+                <div class="header__action">
+                    <!-- <button class="header__btn button-inherit-parent">
+                        <img class="header__icon" src="/components/header/download.svg" alt="Image">
+                    </button>
+                    <button class="header__btn button-inherit-parent">
+                        <img class="header__icon" src="/components/header/printer.svg" alt="Image">
+                    </button>
+                    <button class="header__btn button-inherit-parent">
+                        <img class="header__icon" src="/components/header/bookmark.svg" alt="Image">
+                    </button> -->
```
Comment (zoef-yt) [inline index-legacy.html:45]: remove commented code.

File: index-legacy.html [hunk 17]
```diff
@@ -0,0 +1,128 @@
...
+                    <div id="drawer__content--card" class="drawer__content">
+                        <!-- <img class="drawer__icon" src="/components/drawer/sent.svg" alt="sent mail"> -->
+                        <p class="drawer__icon-description">Card</p>
+                    </div>
+                    <div id="drawer__content--image" class="drawer__content">
+                        <!-- <img class="drawer__icon" src="/components/drawer/sent.svg" alt="sent mail"> -->
+                        <p class="drawer__icon-description">Image</p>
+                    </div>
+                    <div id="drawer__content--input" class="drawer__content">
+                        <!-- <img class="drawer__icon" src="/components/drawer/sent.svg" alt="sent mail"> -->
```
Comment (zoef-yt) [inline index-legacy.html:84]: remove commented code.

File: styles.css [hunk 18]
```diff
@@ -70,10 +71,12 @@ body{
 
 .header__btn {
     font-size: 100%;
-    width: 3em;
-    height: 3em;
-    border-radius: 50%;
+    /* width: 3em;
+    height: 3em; */
+    /* border-radius: 50%; */
```
Comment (zoef-yt) [inline styles.css]: remove commented code.

**Review by zoef-yt — CHANGES_REQUESTED** [review]

**Review by Omkar-Ghate — APPROVED** [review]
Looks great! Good to merge.

File: components/alert/alertPage.css [hunk 19]
```diff
@@ -0,0 +1,52 @@
...
+    position: relative;
+}
+
+.container__components-holder > div:not(:last-child) {
+    margin-bottom: 1.6rem;
+}
+
+iframe {
+    width: 68rem;
+    height: 22.0rem;
```
Comment (iRohitGaur) [inline components/alert/alertPage.css]: this could be 22rem

**Review by iRohitGaur — APPROVED** [review]
Good to merge 👍🏻

File: components/badge/badgePage.html (hunk 1 above)
Comment (varun13169) [reply components/badge/badgePage.html]: Done.

File: components/badge/badgePage.js (hunk 2 above)
Comment (varun13169) [reply components/badge/badgePage.js]: Done.

File: components/button/buttonPage.html (hunk 4 above)
Comment (varun13169) [reply components/button/buttonPage.html]: Done.

File: components/input/inputPage.html (hunk 14 above)
Comment (varun13169) [reply components/input/inputPage.html]: Done.

File: components/card/cardPage.css (hunk 5 above)
Comment (varun13169) [reply components/card/cardPage.css:28]: Personal preference, I like it more explicit.

File: components/button/buttonPage.html (hunk 3 above)
Comment (varun13169) [reply components/button/buttonPage.html]: Done.

File: index-legacy.html (hunk 17 above)
Comment (varun13169) [reply index-legacy.html:84]: Done from the revamped.
Legacy files will be removed shortly.

File: components/alert/alertPage.css (hunk 19 above)
Comment (varun13169) [reply components/alert/alertPage.css]: Done.

File: home.css (hunk 15 above)
Comment (varun13169) [reply home.css]: Done.

File: index-legacy.html (hunk 16 above)
Comment (varun13169) [reply index-legacy.html:45]: Part of legacy file. will be deleted shortly.

File: styles.css (hunk 18 above)
Comment (varun13169) [reply styles.css]: Done.

File: components/card/cardPage.html (hunk 6 above)
Comment (varun13169) [reply components/card/cardPage.html]: Done.

File: components/card/cardPage.js (hunk 8 above)
Comment (varun13169) [reply components/card/cardPage.js]: removed.

File: components/card/cardPage.html (hunk 7 above)
Comment (varun13169) [reply components/card/cardPage.html]: Removed.

File: components/image/imagePage.html (hunk 9 above)
Comment (varun13169) [reply components/image/imagePage.html]: Done.

File: components/image/imagePage.js (hunk 11 above)
Comment (varun13169) [reply components/image/imagePage.js]: Done.

File: components/image/imagePage.html (hunk 10 above)
Comment (varun13169) [reply components/image/imagePage.html]: Done.

File: components/input/inputPage.css (hunk 12 above)
Comment (varun13169) [reply components/input/inputPage.css:27]: Personal Pref, like it explicit.

File: components/input/inputPage.html (hunk 13 above)
Comment (varun13169) [reply components/input/inputPage.html]: Done.

**Pull request merged**